src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerGetTranscriptionWorkflowsIntTest.java

```diff
@@ -153,32 +157,4 @@ void getTranscriptionWorkflowAsNonSuperUserShouldThrow403Error() throws Exceptio
 
         JSONAssert.assertEquals(expectedResponse, actualResponse, JSONCompareMode.NON_EXTENSIBLE);
     }
-
-    private void createWorkflowEntity(TranscriptionEntity transcription,
-                                                             OffsetDateTime workflowTimestamp,
-                                                             TranscriptionStatusEntity status,
-                                                             String comment) {
-
-        TranscriptionWorkflowEntity workflowEntity = new TranscriptionWorkflowEntity();
-        workflowEntity.setTranscription(transcription);
-        workflowEntity.setWorkflowActor(transcription.getCreatedBy());
-        workflowEntity.setWorkflowTimestamp(workflowTimestamp);
-        workflowEntity.setTranscriptionStatus(status);
-
-        dartsDatabase.save(workflowEntity);
-
-        addCommentToWorkflow(workflowEntity, comment, transcription.getCreatedBy());
-    }
-
-    private void addCommentToWorkflow(TranscriptionWorkflowEntity workflowEntity, String comment, UserAccountEntity userAccount) {
-        TranscriptionCommentEntity commentEntity = new TranscriptionCommentEntity();
-        commentEntity.setTranscription(workflowEntity.getTranscription());
-        commentEntity.setTranscriptionWorkflow(workflowEntity);
-        commentEntity.setComment(comment);
-        commentEntity.setCommentTimestamp(workflowEntity.getWorkflowTimestamp());
-        commentEntity.setAuthorUserId(userAccount.getId());
-        commentEntity.setLastModifiedBy(userAccount);
-        commentEntity.setCreatedBy(userAccount);
-        dartsDatabase.save(commentEntity);
-    }
```

Comment:
author note:
I've moved these 2 methods into the transcription stub, reusing existing variants when possible